source/UIAHandler/__init__.py

```diff
@@ -1148,8 +1148,8 @@ def _isUIAWindowHelper(self, hwnd: int, isDebug=False) -> bool:  # noqa: C901
 			try:
 				isOfficeApp = appModule.productName.startswith(("Microsoft Office", "Microsoft Outlook"))
 				isOffice2013OrOlder = int(appModule.productVersion.split(".")[0]) < 16
-			except Exception:
-				log.debugWarning("Unable to parse productName or productVersion, version information likely unavailable", exc_info=True)
+			except RuntimeError:
+				log.debugWarning("Failed parsing productName / productVersion, versionInfo likely missing", exc_info=True)

```

Comment:
`versionInfo` is a module in NVDA. I don't think this abbreviation makes sense here. If you want to fix the lint, break the content into new lines.

```suggestion
				log.debugWarning(
					"Failed parsing productName / productVersion, version information likely missing",
					exc_info=True
				)
```